src/VisualStudio/Xaml/Impl/Features/Commands/XamlEventDescription.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.VisualStudio.LanguageServices.Xaml.Features.Commands
+{
```

Comment:
Feels like this should live under Completion and not Commands. We just happen to pass it around as a command parameter.

Reply:
Yeah I have been moving this struct between Commands and Completion too. Now as you vote for completion I guess completion wins. :)